programs/issue-4-currency-converter/test_currency_converter.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from unittest import TestCase
+from run import currency_converter
+
+
+class TestCurrencyConverter(TestCase):
+    def test_currency_converter(self):
+        start, end, currency = "PLN", "EUR", 1
+        param = {
+            "base": start
+        }
+        response = requests.get("https://api.exchangeratesapi.io/latest", params=param)
```

Comment:
this tests should test `currency_converter`, and not make a request to `https://api.exchangeratesapi.io/latest` API.

You should try to mock a response of this API, by mocking response of requests.get. Here is an example how you can do that: https://realpython.com/testing-third-party-apis-with-mocks/ (don't bother with `nose` in this article - an alternative python testrunner)

TL;DR; Your test by mocking a response of `requests.get` will simulate a server `https://api.exchangeratesapi.io/latest`.

Pros of this approach:
- This will be an unittest and not an integration test
- it will "freeze" in a test in some kind of documentation a response of an API, which current implementation support.
- such unittests will pass on computer being offline. No internet connection is needed.
- they will be much faster - no latency.